k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageViewFragment.java

```diff
@@ -357,20 +382,10 @@ private void messageLoaded(LocalMessage message) {
     }
 
     private void decryptedMessageLoaded(DecryptedMessageLoaded messageViewState) {
-       messageLoaded(messageViewState.getMessage());
-        if (messageViewState.getMoveToSuspiciousFolder()) {
```

Comment:
we remove this check because even if the message is moved to suspicious folder you keep loading it now, right?

Reply:
I removed the check because now I move the messsage to the Suspicious folder in the MessagingRepository. Moving it in the fragment was an error since the fragment can be closed.